m4/libmesh_compiler_control_args.m4

```diff
@@ -0,0 +1,71 @@
+AC_DEFUN([LIBMESH_COMPILER_CONTROL_ARGS],
```

Comment:
Should we call this LIBMESH_ rather than a more general name if we're going to try to use the same logic (the same m4 file, right?) in TIMPI?

Reply:
Yea this does look like it could be plugged directly into libMesh. I'll remove the prefix